app/build.gradle

```diff
@@ -1,8 +1,8 @@
 apply plugin: 'com.android.application'
+def gservices = 'yes'
 
 android {
 	compileSdkVersion 29
-	buildToolsVersion '28.0.3'
```

Comment:
Why is this line gone?

Reply:
IT's in the description

WARNING: The specified Android SDK Build Tools version (28.0.3) is ignored, as it is below the minimum supported version (29.0.2) for Android Gradle Plugin 4.0.1.
Android SDK Build Tools 29.0.2 will be used.
To suppress this warning, remove "buildToolsVersion '28.0.3'" from your build.gradle file, as each version of the Android Gradle Plugin now has a default version of the build tools.